miniapps/tools/convert-dc.cpp

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+//
+//      ----------------------------------------------------------------
+//      Convert DC : convert between different types of data collections
+//      ----------------------------------------------------------------
+//
+// This tool demonstrates how to convert between MFEM's different concrete
+// DataCollection options.
+//
+// Currently supported data collection type options:
+//    visit:                VisItDataCollection (default)
+//    sidre or sidre_hdf5:  SidreDataCollection
+//    json:                 ConduitDataCollection w/ protocol json
+//    conduit_json:         ConduitDataCollection w/ protocol conduit_json
+//    conduit_bin:          ConduitDataCollection w/ protocol conduit_bin
+//    conduit_hdf5:         ConduitDataCollection w/ protocol hdf5
+//
+// Compile with: make convert-dc
+//
+// Serial sample runs:
+//    convert-dc -s ../../examples/Example5 -st visit -o Example5_Conduit -ot json
```

Comment:
Does `convert-dc -s ../../examples/Example5 -o Example5_test` work for you (without an `./Example5` directory)? I think there may be an issue with `prefix_path` in `DataCollection`...

Reply:
yes, this fails for me as well -- I'll look into setting the `prefix_path` correctly

Reply:
I did fix some issue with prefix_path in the visit-dc branch. I think I had to split the collection name into prefix + name in the constructor.